fvtest/compilertest/build/Makefile

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2000, 2016 IBM Corp. and others
+# Copyright (c) 2000, 2018 IBM Corp. and others
```

Comment:
Better not bump the date if we didn't make any other changes?

Reply:
Agreed I'll fix that.